sdk/textanalytics/Azure.AI.TextAnalytics/src/Generated/Models/LinkedEntity.cs

```diff
@@ -11,6 +11,7 @@
 namespace Azure.AI.TextAnalytics
 {
     /// <summary> The LinkedEntity. </summary>
+    /// This property only applies for <see cref="TextAnalyticsClientOptions.ServiceVersion.V3_1"/>, <see cref="TextAnalyticsClientOptions.ServiceVersion.V2022_04_01_Preview"/>, and up.
```

Comment:
I don't think I can add a comment in a Generated file

Reply:
There is a custom file for this class. See => https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/src/LinkedEntity.cs#L67